pkg/provider/provider.go

```diff
@@ -224,6 +225,9 @@ func buildTestEnvironment() { //nolint:funlen
 	env = TestEnvironment{}
 
 	env.variables = *configuration.GetTestParameters()
+	if flags.ConfigurationFile != "" {
```

Comment:
If the goal is to give precedence to flags, why not load environement variables first, then we won't need this code.

Reply:
We could do that but in that case the new "cobra" flags would have to be loaded inside the "certsuite" package or the code to load the env vars moved outside of it. As we agreed to have only flags and no env vars consider this code temporary until the env vars are not used. At the moment it is required for compatibility between both modes of running the test suite.